ddtrace/internal/flare/_subscribers.py

```diff
@@ -61,18 +66,16 @@ def _get_data_from_connector_and_exec(self):
                         "There is already a tracer flare job started at %s. Skipping new request.",
                         str(self.current_request_start),
                     )
-                    return
-                self.current_request_start = datetime.now()
+                    continue
+                if _prepare_tracer_flare(self.flare, configs):
```

Comment:
Don't want to set a timestamp if we just got an agent flare request, not a tracer flare request